test/unit_test/slates_parser_test.cc

```diff
@@ -179,44 +176,42 @@ BOOST_AUTO_TEST_CASE(slates_cache_slot_label)
 
   parser p{8 /*ring_size*/, false /*strict parse*/};
 
-  auto lp = VW::slates::slates_label_parser;
   auto label = scoped_calloc_or_throw<VW::slates::label>();
-  parse_label(lp, &p, "slates slot 0:0.5,1:0.25,2:0.25", *label.get());
-  lp.cache_label(label.get(), io_writer);
+  parse_slates_label(&p, "slates slot 0:0.5,1:0.25,2:0.25", *label.get());
```

Comment:
does a unit test where we init a label but verify the rest of the fields make sense? what is the expectation of the state of the rest of the labels? (just started reading cr, so I might find out as I read)

Reply:
The label is memset to 0 during construction, and because of how it worked in the past that is a valid initial state.